src/Operations.jl

```diff
@@ -2214,6 +2214,9 @@ function status_ext_info(pkg::PackageSpec, env::EnvCache)
             # Check if deps are loaded
             extdeps_info= Tuple{String, Bool}[]
             for extdep in extdeps
+                haskey(weakdepses, extdep) ||
+                    pkgerror(if !isnothing(pkg.name) "$(pkg.name) has a" else "" end,
+                             "malformed Project.toml, the extension package $extdep is not listed in [weakdeps]")
```

Comment:
Very minor but personally I find this kind of one line `if` harder to read than the suggestion.
I also fixed the capitalization at the start.

```suggestion
                    pkgerror(isnothing(pkg.name) ? "M" : "$(pkg.name) has a m",
                             "alformed Project.toml, the extension package $extdep is not listed in [weakdeps]")
```